.github/workflows/release.yaml

```diff
@@ -0,0 +1,23 @@
+name: release
+on:
+  release:
+    types: [created]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Read .nvmrc
+        run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+        id: nvm
+      - name: Use Node.js ${{ steps.nvm.outputs.NVMRC }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: '${{ steps.nvm.outputs.NVMRC }}'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Install dependencies
+        run: npm ci
+      - name: Publish package
+        run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
@cedpoilly, this workflow will build and publish the package to npm when new GitHub releases are created using the version defined in `package.json`.

In order for this to work, we'll need to configure a publish-type npm access token as a [repository secret](https://github.com/vue-gapi/vue-gapi/settings/secrets/actions) named `NPM_TOKEN`.

See the following articles for more information:

* [Creating and viewing access tokens](https://docs.npmjs.com/creating-and-viewing-access-tokens)
* [Creating and viewing access tokens](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository)

Reply:
The release workflow LGTM!

Reply:
FYI – I configured the `NPM_TOKEN` repository secret, cut [v0.4.0](https://github.com/vue-gapi/vue-gapi/releases/tag/v0.4.0), and the [workflow](https://github.com/vue-gapi/vue-gapi/runs/1508075924?check_suite_focus=true) worked like a charm 🎉